components/test_raftstore/src/pd.rs

```diff
@@ -1433,7 +1439,11 @@ impl PdClient for TestPdClient {
         let cluster = self.cluster.rl();
         match cluster.get_region_by_id(region_id) {
             Ok(resp) => {
-                let leader = cluster.leaders.get(&region_id).cloned().unwrap_or_default();
+                let leader = if self.trigger_leader_info_loss.load(Ordering::SeqCst) {
```

Comment:
question: in real world, how does PD have the region information, but does not have leader information?

Reply:
Leader information is not be persisted.